pymc3/distributions/dist_math.py

```diff
@@ -46,7 +46,7 @@
 }
 
 
-def bound(logp, *conditions, **kwargs):
+def bound(logp, *conditions, broadcast_conditions, **kwargs):
```

Comment:
```suggestion
def bound(logp, *conditions, broadcast_conditions=True, **kwargs):
```
That was the old default

Reply:
Yeah good point I'll add that in